test/boilerplate_suggestors/advanced_props_and_state_class_migrator_test.dart

```diff
@@ -173,15 +173,11 @@ void advancedPropsAndStateClassMigratorTestHelper({
           expectedOutput: '''
```

Comment:
The changes in this file should be just:
- FIXME comment updates
- Removal of tests related to mixins from external libs